lib/heroicons/live_view_components.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Heroicons.LiveComponent do
+  use Phoenix.LiveComponent
+  @moduledoc """
+  This module will provide a LiveComponent render helper for rendering Heroicons in a component to avoide boilerplate html code.
+
+  The simplest way to get a Heroicon in your LiveView is:
+
+      <%= live_component @socket, Heroicons.LiveComponent, type: :outline, icon: :server, class: "ml-6 w-12 h-12 text-green-400" %>
+
+  
+  """
+  @moduledoc since: "0.3.0"
+
+  @callback render(assigns :: Phoenix.LiveView.Socket.assigns()) ::  Phoenix.LiveView.Rendered.t()
+  @doc false
+
+
+  def render(%{type: :outline, class: class, icon: icon} = assigns) do
+    ~L"""
+    <%=  apply(Elixir.Heroicons.Outline, icon, [%{class: class}]) |> Phoenix.HTML.raw %>
```

Comment:
I think the functions return html safe data, is the call to `raw` necessary?

Reply:
good point - you are right - I updated this part.